install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -71,6 +71,11 @@ node with `*Key=KubernetesCluster,Value=clusterid*`.
 |xref:../../install_config/configuring_openstack.adoc#install-config-configuring-openstack[Configuring for OpenStack]
 |
 
+|Trident from NetApp
+|`netapp.io/trident`
+|xref:../../install_config/configuring_trident.adoc[Configuring for Trident]
```

Comment:
What you can do is point to this link instead of removing it: https://github.com/NetApp/trident

All of the instructions about how to get that going are there.